templates/devfile.yaml

```diff
@@ -0,0 +1,83 @@
+schemaVersion: 2.0.0
```

Comment:
Can we put the copyright in this file too?

Reply:
Not sure exactly where we draw the line.. maybe:

buildfiles - pom.xml / build.gradle - maybe
other XML - server.xml - maybe - if significant
devfile.yaml - yes - since it's the central artifact here
Dockerfiles - no.. most seem to not have them?